Makefile

```diff
@@ -23,7 +23,10 @@ wait-remove:
 
 remove:
 # remove app from current model
-	juju remove-application --destroy-storage $(app)
+	juju remove-application $(app)
+	juju status --storage --format=json 2>/dev/null \
+		| jq -r '.storage.volumes[] | select( .status.current == "detached" ) | .storage' \
+		| xargs juju remove-storage
```

Comment:
out of curiosity, what is the difference between this and `juju remove-application --destroy-storage $(app)`?

Reply:
for unknown reason some versions of juju stuck with `--destroy-storage`